### PR TITLE
Refactors some mobcode

### PR DESCRIPTION
**Pull request by ChangelingRain**
Does nothing to gameplay at all anymore!

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/interaction.dm [hunk 1]
```diff
@@ -103,6 +108,16 @@
 	else
 		..()
 
+/mob/living/simple_animal/drone/getarmor(def_zone, type)
+	var/armorval = 0
+
+	if(istype(head, /obj/item/clothing) && islist(head.armor))
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/friendly/drone/interaction.dm]: You no longer need to do the istype or islist checks, all /obj have the armor list var.

**Comment by PKPenguin321** [comment]
👎 drones should not be any harder to kill

**Comment by ExcessiveUseOfCobblestone** [comment]
drones should have a special killword associated with them the RD can tell everyone so when someone gets mad at a drone they can just say the killword ala deus ex

**Comment by GunHog** [comment]
> drones should have a special killword associated with them that the RD can tell everyone so when someone gets mad at a drone they can just say the killword ala deus ex

If there is an RD, you can just ask him to kill the drones, as he can remotely gib them from his robotics console.

**Comment by Shadowlight213** [comment]
Yeah. Console gib works on both living and dead drones.

**Comment by optimumtact** [comment]
@KorPhaeron y/n ? I'm against it personally

**Comment by KorPhaeron** [comment]
I don't really care and our glorious leader Cheridan downvoted it

**Comment by ChangelingRain** [comment]
I'm going to pretend this is a new pr ignore everything that doesn't suggest that

**Pull request merged**